enterprise/frontend/src/embedding-sdk/cli/utils/get-package-version.ts

```diff
@@ -11,28 +11,57 @@ export const hasPackageJson = async () => {
   }
 };
 
-export const getPackageVersion = async (packageName: string) => {
+type DependencyMap = Record<string, string>;
+
+export const getPackageVersions = async (...packageNames: string[]) => {
+  const versions: Record<string, string> = {};
+
+  const projectDeps = await getProjectDependenciesFromPackageJson();
+
+  for (const name of packageNames) {
+    versions[name] =
+      (await getPackageVersionFromModule(name)) || projectDeps?.[name] || null;
```

Comment:
This makes it so that we have the exact version of the dependencies to show to the user in the CLI (e.g. 18.2.3) instead of a rough semver range (e.g. ^18.4). It also prevents us from printing `file:../../` version in development.